tests/testthat/_snaps/tune_args.md

```diff
@@ -0,0 +1,23 @@
+# tune_args() errors on multiple tune()s in same arg
+
+    Code
+      recipe(~., data = mtcars) %>% step_pca(all_predictors(), num_comp = ~ tune() +
+        tune()) %>% tune_args()
+    Condition
+      Error in `purrr::map()`:
+      i In index: 1.
+      Caused by error in `purrr::map_chr()`:
+      i In index: 1.
+      i With name: num_comp.
```

Comment:
I thiiink this is the same place as what I called out before, but would be awesome to trim these two layers of rethrowing out of the message.

Reply:
fixed in [`b6fd10b` (#1388)](https://github.com/tidymodels/recipes/pull/1388/commits/b6fd10b36a789826625e1e64f656e7fe84aa0ae9)